internal/quaiapi/quai_api.go

```diff
@@ -658,7 +658,7 @@ func (s *PublicBlockChainQuaiAPI) CreateAccessList(ctx context.Context, args Tra
 	if !s.b.ProcessingState() {
 		return nil, errors.New("createAccessList call can only be made on chain processing the state")
 	}
-	bNrOrHash := rpc.BlockNumberOrHashWithNumber(rpc.PendingBlockNumber)
```

Comment:
Wouldn't we prefer pending block number to simulate an access list for the next block about to be made if storage slots are potentially block number specific?

Reply:
Pending header is returning -2

Reply:
The pending block number is -2 and results in no block found